app/client/Widgets/NothingPlayingWidget.h

```diff
@@ -43,6 +43,9 @@ class NothingPlayingWidget : public QFrame
 private slots:
     void onSessionChanged( const unicorn::Session& session );
 
+#if defined( Q_OS_MAC )
+    void onAppleMusicClicked();
```

Comment:
We probably still want `void onAppleMusicClicked();`  for `Q_OS_WIN` so that nothing fires?